cob_twist_controller/include/cob_twist_controller/inverse_differential_kinematics_solver.h

```diff
@@ -73,8 +73,11 @@ class InverseDifferentialKinematicsSolver
         this->kinematic_extension_.reset(KinematicExtensionBuilder::createKinematicExtension(this->params_));
         this->limiter_params_ = this->kinematic_extension_->adjustLimiterParams(this->limiter_params_);
 
-        this->limiters_.reset(new LimiterContainer(this->limiter_params_));
+        this->limiters_.reset(new LimiterJointContainer(this->limiter_params_));
         this->limiters_->init();
+
+        this->cartesian_limiters_.reset(new LimiterCartesianContainer(this->limiter_params_));
+        this->cartesian_limiters_->init();
```

Comment:
I'd go for naming the LimiterJointContainer `output_limiters_`
and the LimiterCartesianContainer `input_limiters_`